src/tests/EdgeID/transition_id_tests.rs

```diff
@@ -0,0 +1,120 @@
+        vec![
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E43".to_string())),
+                Box::new(TransitionID::Simple("E31".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E37".to_string())),
+                Box::new(TransitionID::Simple("E34".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E42".to_string())),
+                Box::new(TransitionID::Simple("E33".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E37".to_string())),
+                Box::new(TransitionID::Simple("E35".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E42".to_string())),
+                Box::new(TransitionID::Simple("E30".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E39".to_string())),
+                Box::new(TransitionID::Simple("E31".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E38".to_string())),
+                Box::new(TransitionID::Simple("E32".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E41".to_string())),
+                Box::new(TransitionID::Simple("E34".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E40".to_string())),
+                Box::new(TransitionID::Simple("E33".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E40".to_string())),
+                Box::new(TransitionID::Simple("E30".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E38".to_string())),
+                Box::new(TransitionID::Simple("E36".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E41".to_string())),
+                Box::new(TransitionID::Simple("E35".to_string()))
+            )
+            ]; "Conjunction HalfAdm1 and HalfAdm2")]
+            #[test_case(FOLDER_PATH, QueryExpression::Quotient(
+                Box::new(QueryExpression::VarName("HalfAdm1".to_string())),
+                Box::new(QueryExpression::VarName("HalfAdm2".to_string()))
+            ), 
+            vec![
+              TransitionID::Simple("E25".to_string()), 
+              TransitionID::Simple("E26".to_string()), 
+              TransitionID::Simple("E27".to_string()), 
+              TransitionID::Simple("E28".to_string()), 
+              TransitionID::Simple("E29".to_string())]; "Quotient HalfAdm1 and HalfAdm2")]
+    fn transition_id_test(
```

Comment:
I feel a better and informed function name is needed.

Reply:
renamed to tranistion_id_test